packages/client/components/NewMeetingActions.tsx

```diff
@@ -65,19 +71,37 @@ const NewMeetingActions = (props: Props) => {
       fragment NewMeetingActions_team on Team {
         ...NewMeetingActionsCurrentMeetings_team
         id
+        organization {
+          name
+          lockedAt
+        }
       }
     `,
     teamRef
   )
 
+  const {organization} = team
+
+  const {lockedAt, name: organizationName} = organization
+
+  const isLocked = !!lockedAt
+
   return (
     <ActionRow>
+      {error && <ErrorBlock>{error.message}</ErrorBlock>}
+      {isLocked && (
+        <ErrorBlock>
+          Unfortunately, {organizationName} has exceeded the {Threshold.MAX_PERSONAL_TIER_TEAMS}{' '}
+          teams limit on the Starter Plan for more than {Threshold.PERSONAL_TIER_LOCK_AFTER_DAYS}{' '}
+          days, and your account has been locked. You can re-activate your teams by upgrading your
+          account.
+        </ErrorBlock>
+      )}
```

Comment:
-1 Can we add an `Upgrade CTA Viewed` event when this error block is returned?  We also need to add this location to `UpgradeCTALocationEnumType`.

Reply:
Added